src/sentry_plugins/bitbucket/client.py

```diff
@@ -10,33 +13,38 @@
 
 
 class BitbucketClient(object):
-    API_URL = u'https://api.bitbucket.org/1.0'
+    API_URL = u'https://api.bitbucket.org/'
 
-    def __init__(self, auth):
+    def __init__(self, auth=None):
         self.auth = auth
 
-    def request(self, method, path, data=None, params=None):
-        oauth = OAuth1(unicode(settings.BITBUCKET_CONSUMER_KEY),
-                       unicode(settings.BITBUCKET_CONSUMER_SECRET),
+    def request(self, method, version, path, data=None, params=None, json=True):
+        oauth = OAuth1(six.text_type(settings.BITBUCKET_CONSUMER_KEY),
+                       six.text_type(settings.BITBUCKET_CONSUMER_SECRET),
                        self.auth.tokens['oauth_token'], self.auth.tokens['oauth_token_secret'],
```

Comment:
it looks like auth is not required, yet this will hard error whenever its not present (KeyError)